index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- [ ] Please don't merge the pull request until the reviewer approves it, create a new open pull request